app/models/event_topic.py

```diff
@@ -13,22 +14,34 @@ def get_new_slug(name):
         return '{}-{}'.format(slug, uuid.uuid4().hex)
 
 
+default_image = current_app.config.get('SERVER_NAME') + '/static/header.png'
```

Comment:
@SaptakS @iamareebjamal Can you help with this.
https://travis-ci.org/fossasia/open-event-server/builds/388320978

Reply:
You cannot access `app` outside of context

Reply:
Please tell how to deal as this PR is open from very long.
How to modify it.
`default_image = current_app.config.get('SERVER_NAME') + '/static/header.png' `

Reply:
Use this inside a method as not as a static constant